<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddStudentDialog</class>
 <widget class="QDialog" name="AddStudentDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>334</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">

/* === QLineEdit –ø–æ–ª—è === */
QLineEdit {
    background-color: #ffffff;
    border: 1px solid #cccccc;
    border-radius: 6px;
    padding: 6px 8px;
    selection-background-color: #a8d0ff;
}



/* === QPushButton === */
QPushButton {
    background-color: #4a90e2;
    color: white;
    border: none;
    border-radius: 6px;
    padding: 8px 16px;
}

QPushButton:hover {
    background-color: #357abd;
}

QPushButton:pressed {
    background-color: #2a5d9f;
}

/* === QDialogButtonBox –∫–Ω–æ–ø–∫–∏ === */
QDialogButtonBox QPushButton {
    min-width: 80px;
}

/* === QLabels === */
QLabel {
    font-weight: 500;
}

/* === QLineEdit –ø–æ–ª—è (—Ç—ë–º–Ω–∞—è —Ç–µ–º–∞) === */
QLineEdit {
    background-color: #2b2b2b;    /* —Ç—ë–º–Ω—ã–π —Ñ–æ–Ω */
    color: #ffffff;               /* –±–µ–ª—ã–π —Ç–µ–∫—Å—Ç */
    border: 1px solid #555555;    /* —Ç—ë–º–Ω–∞—è –≥—Ä–∞–Ω–∏—Ü–∞ */
    border-radius: 6px;
    padding: 6px 8px;
    selection-background-color: #4a90e2; /* —Ü–≤–µ—Ç –≤—ã–¥–µ–ª–µ–Ω–∏—è */
    selection-color: #ffffff;             /* —Ü–≤–µ—Ç —Ç–µ–∫—Å—Ç–∞ –ø—Ä–∏ –≤—ã–¥–µ–ª–µ–Ω–∏–∏ */
}

/* === –≠—Ñ—Ñ–µ–∫—Ç –ø—Ä–∏ —Ñ–æ–∫—É—Å–µ === */
QLineEdit:focus {
    border: 2px solid #4a90e2;
    outline: none;
}</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>471</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="lineEditFile">
      <property name="placeholderText">
       <string>–§–∞–π–ª —Å–æ—Ö—Ä–∞–Ω–µ–Ω–∏—è</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditData">
      <property name="placeholderText">
       <string>–î–∞–Ω–Ω—ã–µ —Ñ–æ—Ä–º–∞—Ç–µ –§–ò–û –ë–ê–õ–õ</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnBrowse">
      <property name="text">
       <string>–í—ã–±–æ—Ä üìÇ</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnOk">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>250</y>
     <width>90</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>–û–∫</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClose">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>250</y>
     <width>90</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>–ó–∞–∫—Ä—ã—Ç—å</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
